code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -1,67 +1,81 @@
-// robot_upgrades.dm
-// Contains various borg upgrades.
-
+/*
+ * Cyborg/Robot Upgrades
+ *
+ * Contains various robot upgrade modules.
+ */
 /obj/item/borg/upgrade
 	name = "borg upgrade module."
 	desc = "Protected by FRM."
 	icon = 'icons/obj/module.dmi'
 	icon_state = "cyborg_upgrade"
-	var/locked = 0
-	var/require_module = 0
-	var/installed = 0
 
-/obj/item/borg/upgrade/proc/action(var/mob/living/silicon/robot/R)
+	var/locked = FALSE
+	var/require_module = FALSE
+	var/installed = FALSE
+
+/obj/item/borg/upgrade/proc/action(mob/living/silicon/robot/R)
 	if(R.stat == DEAD)
-		to_chat(usr, "<span class='warning'>The [src] will not function on a deceased robot.</span>")
-		return 1
-	return 0
+		to_chat(usr, SPAN_WARNING("The [src] will not function on a deceased robot."))
+		return FALSE
 
+	return TRUE
 
+/*
+ * Reset Module
+ */
 /obj/item/borg/upgrade/reset
 	name = "robotic module reset board"
 	desc = "Used to reset a cyborg's module. Destroys any other upgrades applied to the robot."
 	icon_state = "cyborg_upgrade1"
 	item_state = "cyborg_upgrade"
-	require_module = 1
+	require_module = TRUE
+
+/obj/item/borg/upgrade/reset/action(mob/living/silicon/robot/R)
+	if(!..())
+		return FALSE
 
-/obj/item/borg/upgrade/reset/action(var/mob/living/silicon/robot/R)
-	if(..())
-		return 0
 	R.module_reset()
-	return 1
+	return TRUE
 
+/*
+ * Rename Module
+ */
 /obj/item/borg/upgrade/rename
 	name = "robot reclassification board"
 	desc = "Used to rename a cyborg."
 	icon_state = "cyborg_upgrade1"
 	item_state = "cyborg_upgrade"
+
 	var/heldname = "default name"
 
 /obj/item/borg/upgrade/rename/attack_self(mob/user as mob)
 	heldname = sanitizeSafe(input(user, "Enter new robot name", "Robot Reclassification", heldname), MAX_NAME_LEN)
 
-/obj/item/borg/upgrade/rename/action(var/mob/living/silicon/robot/R)
-	if(..()) return 0
+/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/R)
+	if(!..())
+		return FALSE
+
 	R.notify_ai(ROBOT_NOTIFICATION_NEW_NAME, R.name, heldname)
 	R.name = heldname
 	R.custom_name = heldname
 	R.real_name = heldname
+	return TRUE
 
-	return 1
-
+/*
+ * Restart Module
+ */
```

Comment:
I won't ask you to remove them, but in my opinion none of these need a comment saying what's said two lines below.

Aside, avoid using block comments for trivial things. If you want visually better-separated concerns, double newline between them and be cagey about adding any singles as well.

Reply:
But I will!
Files that get too long are a pet peeve of mine, and needless comments (especially needless block comments) are a very good way to reach that state.

Comments should explain concisely what the code cannot. Restating the obvious is wasteful.
Within the scope of ss13 classes, either the type or the name or the description should plainly describe what a thing is. File purpose should be encapsulated by the filename, but I'm not so hostile towards file headers.